payments-core/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -241,7 +243,10 @@ sealed interface StripeIntent : StripeModel {
             }
         }
 
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
         @Parcelize
-        internal data class WeChatPayRedirect(val weChat: WeChat) : NextActionData()
+        data class WeChatPayRedirect(val weChat: WeChat) : NextActionData()
```

Comment:
does the property need to be public?

Reply:
Yes, it needs to be downcasted in `WeChatPayAUthenticator` in order to extract the `WeChat` object